templates/default/node/nodeeditbox.html

```diff
@@ -158,10 +158,27 @@
 										<input type="hidden" name="nodeedit[location_street]" value="{$nodeinfo.location_street}">
 										<input type="hidden" name="nodeedit[location_house]" value="{$nodeinfo.location_house}">
 										<input type="hidden" name="nodeedit[location_flat]" value="{$nodeinfo.location_flat}">
-										<input type="checkbox" name="nodeedit[teryt]" id="teryt" onchange="check_teryt('location')"{if $nodeinfo.teryt} checked{/if}><label for="teryt">{trans("TERRIT-DB")}</label>
+										<input type="checkbox" name="nodeedit[teryt]" id="teryt" onchange="check_teryt('location')"{if $nodeinfo.teryt} checked{/if}> <label for="teryt">{trans("TERRIT-DB")}</label>
 									</td>
 								</tr>
 
+									<tr>
```

Comment:
Czyli reasumując, jeśli ma być przypisywanie do grupy to tylko w nodeadd?
W nodeedit nie było przypisywania i tak ma zostać?